seed/seed.js

```diff
@@ -1,7 +1,7 @@
 const faker = require('faker');
 const fs = require('fs');
 const pg = require('pg');
-// const copyFrom = require('pg-copy-streams').from;
+const copyFrom = require('pg-copy-streams').from;
 
 const recipeRow = function recipeRow(id) {

```

Comment:
I don't think that it's necessary to repeat the function name here since it would be a violation of the DRY principle